services/repository/contributors_graph.go

```diff
@@ -90,7 +91,7 @@ func generateContributorStats(genDone chan struct{}, cache cache.Cache, cacheKey
 	gitRepo, closer, err := git.RepositoryFromContextOrOpen(ctx, repo.RepoPath())
 	if err != nil {
 		err := fmt.Errorf("OpenRepository: %w", err)
-		_ = cache.Put(cacheKey, err, contributorStatsCacheTimeout)
+		cache.Put(cacheKey, err, contributorStatsCacheTimeout)
```

Comment:
This one get reverted too. I don't know the effect of assigning it to variable. Is this important?